src/guides/v2.3/config-guide/cli/config-cli-subcommands-index.md

```diff
@@ -123,6 +123,47 @@ Catalog Search index has been rebuilt successfully in <time>
 {:.bs-callout-info}
 Reindexing all indexers can take a long time for stores with large numbers of products, customers, categories, and promotional rules.
 
+### Reindex in parallel mode
+
+As of Magento 2.2.6 or later, Indexers are scoped and multi-threaded to support reindexing in parallel mode. This feature reduces processing time. It parallelizes by the indexer’s dimension and executes across multiple threads.
+
+Index parallelization can affect only the indexers which are scoped, which means Magento split the data into multiple tables by indexer as its scope instead of keeping all data in one table.
```

Comment:
Perhaps this could explained a bit more. Define 'scoped' a bit and how that determines how the data is broken up.